### DISCUSSION TITLE
Home Assistant und Smart Appliance Enabler

### DISCUSSION THREAD

**Discussion by Bahnhof100** [camueller/SmartApplianceEnabler#505] [Q&A]
Ich bin neu hier. Und lese mich gerade in das Projekt ein. Zuallererst finde ich die Idee super!
Nun habe ich an das Forum einige Fragen.
Da ich meine Hausautomation mit Homeassitant realisiert habe und Zusätzlich meine Daten an das Sunny Portal sende will. Gibt es schon eine einfache Integration in Homeassitant? Oder muss man das über Mptt, ibrocker machen? Wenn ja benötige ich da ein zweites PI?

**Comment by andi5783** [comment]
Hi und willkommen,
einfach [das hier](https://github.com/camueller/SmartApplianceEnabler) lesen, dann weiß man was der SAE kann. Der SAE ist eine Standalone Lösung die über verschiedene Protokolle kommunizieren kann.

**Comment by TJ23456** [comment]
Hallo,
Ähnlich wie "Bahnhof100" nutze ich auch Homeassistant und würde auch gern per RestAPI z.B. meine Zigbee-Plugs oder -DIN-Rail-Schalter messen und steuern.

Soweit ich das überblicke, wird dafür in SAE nur noch ein "Bearer"-Token für HTTP-Requests (statt Benutzername/Passwort) in der Zähler+Schalter-GUI benötigt. Das Bearer-Token kann ja schon im Script "notifyWithHA.sh" verwendet werden.

Eine "Lösung" mittels ReverseProxy finde ich eher aufwändig/umständlich...
